src/main/java/org/openjdk/jextract/clang/libclang/RuntimeHelper.java

```diff
@@ -62,11 +62,9 @@ private RuntimeHelper() {}
             (size, align) -> MemorySegment.allocateNative(size, align, MemorySession.openImplicit());
 
     static {
-        // Manual change to handle platform specific library name difference
-        String libName = System.getProperty("os.name").startsWith("Windows")? "libclang" : "clang";
-        System.loadLibrary(libName);
-
-        SYMBOL_LOOKUP = name -> SymbolLookup.loaderLookup().lookup(name).or(() -> LINKER.defaultLookup().lookup(name));
+        System.loadLibrary("clang");
```

Comment:
This needs the fix for Windows

Reply:
oops. missed that manual change. will fix it.